src/app/components/layout/navbar/navbar.component.ts

```diff
@@ -155,34 +156,31 @@ export class NavbarComponent implements OnInit, AfterViewInit, OnDestroy {
           northboundServices = sortBy(northboundServices, function (obj) {
             return obj.name.toLowerCase();
           });
-          const notificationService = servicesData.filter((el => (el.type === 'Notification')));
-          const managementService = servicesData.filter((el => (el.type === 'Management')));
-          const dispatcherService = servicesData.filter((el => (el.type === 'Dispatcher')));
-          const bucketStorageService = servicesData.filter((el => (el.type === 'BucketStorage')));
+
+          const currentService = JSON.parse(sessionStorage.getItem('SERVICE_NAME'));
```

Comment:
What does currentService mean?

Reply:
please check below answer

Reply:
My concern is on variable name currentService; is this for the service selected to open modal for?